runtime/gc_include/ObjectAllocationAPI.hpp

```diff
@@ -70,55 +77,58 @@ class MM_ObjectAllocationAPI
 	}
 
 	VMINLINE void
-	initializeContiguousIndexableObject(J9VMThread *currentThread, bool initializeSlots, J9Class *arrayClass, uint32_t size, uintptr_t dataSize, j9object_t *objectHeader)
+	initializeContiguousIndexableObject(J9VMThread *currentThread, bool initializeSlots, J9Class *arrayClass, uint32_t size, uintptr_t dataSize, j9object_t objectHeader)
 	{
+		void *dataAddr = (void *)((uintptr_t)objectHeader + _contiguousHeaderSize);
 		bool isCompressedReferences = J9VMTHREAD_COMPRESS_OBJECT_REFERENCES(currentThread);
-
 		if (isCompressedReferences) {
-			uintptr_t headerSize = sizeof(J9IndexableObjectContiguousCompressed);
-			J9IndexableObjectContiguousCompressed *header = (J9IndexableObjectContiguousCompressed*)*objectHeader;
+			J9IndexableObjectContiguousCompressed *header = (J9IndexableObjectContiguousCompressed*)objectHeader;
```

Comment:
I think the coding standard requires the space in front of '*' even in this case, but please check the guidelines (or the what is more prevalent in the code) and update all the spots if it's the case